Dockerfile-dev

```diff
@@ -5,7 +5,8 @@ WORKDIR /usr/src/love/
 COPY requirements.txt .
 RUN source /opt/lsst/software/stack/loadLSST.bash && \
```

Comment:
Replace 

```
source /opt/lsst/software/stack/loadLSST.bash
```

by 

```
/home/saluser/.setup_dev.sh
```

This will give you all the tssw package, including `ts_utils`. You should be able to remove 

```
pip install /home/saluser/repos/ts_utils
```

if you do that. All the packages in `/home/saluser/repos/` are available as `eups` packages.